package.json

```diff
@@ -63,5 +63,9 @@
     "node": "^6.5.0",
     "npm": "^3.10.3"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "devDependencies": {
```

Comment:
I will change that because I didn't understand the root package stuff in the first place :D I will do the change.